### ISSUE TITLE
Where to see the Joint Accuracy evaluation?

### ISSUE THREAD

**Issue by CaoHoangTung** [SKTBrain/SUMBT#12]
When evaluating the model, I can't see the Exact Joint Accuracy of **0.48806** reported in the paper (both in the output text log and tensorboard log). Is this logged somewhere or do I have to implement it?

Here is the content of eval_results_all.txt, which to my understanding, does not have the exact accuracy score
```
eval_acc_slot = 0.80290287733078	0.867335855960846	0.7904232144355774	1.0	1.0	1.0	1.0	1.0	0.8619099259376526	0.870455801486969	0.8585187196731567	0.8693705797195435	0.8610960245132446	0.8431904315948486	0.8776451349258423	0.8274552226066589	0.8518719673156738	0.804530680179596	0.7505425810813904	0.8275908827781677	0.8244709968566895	0.8273195624351501	0.7339934706687927	0.8518719673156738	0.7593597173690796	0.9686652421951294	0.9226804375648499	0.9221377968788147	0.9564568400382996	0.799782931804657	0.8491590023040771	0.6471785306930542	0.65233314037323	0.6389039754867554	0.8499728441238403
eval_accuracy = 0.8505464778344914
eval_loss = 28.46753237458331
eval_loss_slot = 0.8817319130627069	1.201549073308302	1.2350054645523776	0.036111686570932194	0.005953068851795991	0.011532692755009056	0.007900203865390863	0.14539033580395558	0.6507461597567916	0.48282753164479253	0.5047258990239433	0.48811625681018606	0.3414967832347172	1.1946264176436954	0.357626472429003	0.5301039830917453	0.5166375836485323	0.40926279729041176	0.7328648737402319	0.7531180265048802	0.706450523843719	1.1251465154754132	1.739766523929245	1.3978182440274964	0.7326630629716331	0.4957690410198754	1.0787263601306851	1.1032195673387335	0.6402477735064305	1.523714961210186	0.7175217050459182	1.2880614138179527	1.9950037940455336	2.0547096399975433	1.3813860977934678
loss = None

```
